app/views/sites/_settings_tab.html.erb

```diff
@@ -0,0 +1,32 @@
+        <div role="tabpanel" class="tab-pane active" id="labels">
```

Comment:
When we did tabbed forms in spotlight, we kept the tab panel markup in `sites/edit` to make the connection between the tablist and the panel is easier to see (with the additional benefit of making this form portable to non-tab contexts, should we have any).

Another thing we did was wrap the whole tab business in the form, so a single submit button will submit the data for all tabs (otherwise you end up in a weird situation where you can only fill out one tab at a time).